2021-09-08 13:38:57,023 INFO    MainThread:11691 [wandb_setup.py:_flush():69] setting env: {}
2021-09-08 13:38:57,024 INFO    MainThread:11691 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-08 13:38:57,024 INFO    MainThread:11691 [wandb_init.py:_log_setup():342] Logging user logs to /home/jungwoo/catkin_ws/src/RL_practice/DDPG/wandb/run-20210908_133857-1qv1lgff/logs/debug.log
2021-09-08 13:38:57,024 INFO    MainThread:11691 [wandb_init.py:_log_setup():343] Logging internal logs to /home/jungwoo/catkin_ws/src/RL_practice/DDPG/wandb/run-20210908_133857-1qv1lgff/logs/debug-internal.log
2021-09-08 13:38:57,025 INFO    MainThread:11691 [wandb_init.py:init():375] calling init triggers
2021-09-08 13:38:57,026 INFO    MainThread:11691 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'env': 'Acrobot', 'q_lr': 0.001, 'mu_lr': 0.0001, 'gamma': 0.99, 'tau': 0.001, 'batch_size': 64}
2021-09-08 13:38:57,026 INFO    MainThread:11691 [wandb_init.py:init():424] starting backend
2021-09-08 13:38:57,027 INFO    MainThread:11691 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-08 13:38:57,051 INFO    MainThread:11691 [backend.py:ensure_launched():135] starting backend process...
2021-09-08 13:38:57,059 INFO    MainThread:11691 [backend.py:ensure_launched():139] started backend process with pid: 11701
2021-09-08 13:38:57,060 INFO    MainThread:11691 [wandb_init.py:init():429] backend started and connected
2021-09-08 13:38:57,063 INFO    MainThread:11691 [wandb_init.py:init():477] updated telemetry
2021-09-08 13:38:57,063 INFO    MainThread:11691 [wandb_init.py:init():500] communicating current version
2021-09-08 13:38:57,434 INFO    MainThread:11691 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-08 13:38:57,435 INFO    MainThread:11691 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-08 13:38:57,893 INFO    MainThread:11691 [wandb_init.py:init():540] starting run threads in backend
2021-09-08 13:38:59,442 INFO    MainThread:11691 [wandb_run.py:_console_start():1584] atexit reg
2021-09-08 13:38:59,442 INFO    MainThread:11691 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-09-08 13:38:59,443 INFO    MainThread:11691 [wandb_run.py:_redirect():1463] Redirecting console.
2021-09-08 13:38:59,444 INFO    MainThread:11691 [wandb_run.py:_redirect():1519] Redirects installed.
2021-09-08 13:38:59,444 INFO    MainThread:11691 [wandb_init.py:init():565] run started, returning control to user process
2021-09-08 13:38:59,456 INFO    MainThread:11691 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-09-08 13:38:59,456 INFO    MainThread:11691 [wandb_run.py:_restore():1526] restore
2021-09-08 13:39:01,971 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 802
  total_bytes: 802
}

2021-09-08 13:39:02,384 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 802
  total_bytes: 802
}

2021-09-08 13:39:02,938 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 802
  total_bytes: 5057
}

2021-09-08 13:39:03,042 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 802
  total_bytes: 6144
}

2021-09-08 13:39:03,146 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 802
  total_bytes: 6144
}

2021-09-08 13:39:03,248 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 802
  total_bytes: 6144
}

2021-09-08 13:39:03,350 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 802
  total_bytes: 6144
}

2021-09-08 13:39:03,454 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:03,558 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:03,660 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:03,770 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:03,873 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:03,977 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:04,551 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:04,654 INFO    MainThread:11691 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6144
  total_bytes: 6144
}

2021-09-08 13:39:05,741 INFO    MainThread:11691 [wandb_run.py:_show_files():1898] logging synced files
